src/main/flight/pid_init.c

```diff
@@ -238,6 +238,13 @@ void pidInitFilters(const pidProfile_t *pidProfile)
 
     pt1FilterInit(&pidRuntime.antiGravityThrottleLpf, pt1FilterGain(ANTI_GRAVITY_THROTTLE_FILTER_CUTOFF, pidRuntime.dT));
     pt1FilterInit(&pidRuntime.antiGravitySmoothLpf, pt1FilterGain(ANTI_GRAVITY_SMOOTH_FILTER_CUTOFF, pidRuntime.dT));
+
+#ifdef USE_ACC
+    const float k = pt3FilterGain(100, pidRuntime.dT);
```

Comment:
```suggestion
    #define ATTITUDE_CUTOFF_HZ 100
    const float k = pt3FilterGain(ATTITUDE_CUTOFF_HZ, pidRuntime.dT);
```

Reply:
It is just for testing. I will do it as variable to make it changeable through CLI.

Reply:
Fixed at least using define :)

Reply:
Okay, I see, sorry